File: src/python/core/domain/entities/skill.py
from dataclasses import dataclass
from typing import Optional
from dataclasses_json import dataclass_json


@dataclass_json
@dataclass
class Skill:
    SkillName: str
    SkillLevel: Optional[str] = None
    Years: Optional[float] = None
    Evidence: Optional[str] = None

File: src/dotnet/Core/Domain/Entities/Skill.cs
using System.Text.Json.Serialization;

namespace Rag.Candidates.Core.Domain.Entities;

public class Skill
{
    [JsonPropertyName("SkillName")]
    public string? Name { get; set; }
    
    [JsonPropertyName("SkillCategory")]
    public string? Category { get; set; }
    
    [JsonPropertyName("SkillLevel")]
    public string? Level { get; set; }
    
    [JsonPropertyName("YearsExperience")]
    public int? YearsExperience { get; set; }
    
    [JsonPropertyName("Evidence")]
    public string? Evidence { get; set; }
    
    [JsonPropertyName("IsRelevant")]
    public bool? IsRelevant { get; set; }
}